ts/screens/onboarding/OnboardingCompletedScreen.tsx

```diff
@@ -23,9 +23,8 @@ const OnboardingCompletedScreen = () => {
     <BaseScreenComponent>
       <SafeAreaView style={IOStyles.flex}>
         <InfoScreenComponent
-          image={<Completed width={80} height={80} />}
+          image={<PictogramFireworks size={120} color="#00C5CA" />}
```

Comment:
Here's the correct way to reference that specific pictogram:
```suggestion
          image={<Pictogram name="fireworks" />}
```
There's no need to specify both size and color because those values are set by default.

Reply:
do it